google/cloud/security/scanner/scanners/scanners_map.py

```diff
@@ -28,21 +29,24 @@
     'BucketsRulesEngine': BucketsAclScanner,
     'CloudSqlRulesEngine': CloudSqlAclScanner,
     'GroupsRulesEngine': GroupsScanner,
-    'IamRulesEngine': IamPolicyScanner
+    'IamRulesEngine': IamPolicyScanner,
+    'ForwardingRuleRulesEngine' : ForwardingRuleScanner,
```

Comment:
nit: Can you alphasort, i.e. move ForwardingRuleRulesEngine after CloudSqlRulesEngine